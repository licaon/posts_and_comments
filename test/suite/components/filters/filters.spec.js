import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Filters from 'src/components/filters/filters';

describe('Filters component', () => {
  const createComponent = (props) => {
    const wrapper = shallow(
      <Filters
        {...props}
      />);
    return toJson(wrapper);
  };
  it('should render component correctly', (done) => {
    const component = createComponent();

    expect(component).toMatchSnapshot();
    done();
  });
});
