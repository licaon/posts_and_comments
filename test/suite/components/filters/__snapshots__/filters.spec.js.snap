// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filters component should render component correctly 1`] = `
<div
  className="md-cell"
>
  <Connect(UserFilter) />
  <Connect(Search) />
</div>
`;
